src/keywordPreservation.ts

```diff
@@ -186,21 +202,21 @@ export const preserveKeywords = ({
   localAssets: object;
   remoteAssets: object;
   keywordMappings: KeywordMappings;
-  auth0Handlers: Pick<APIHandler, 'id' | 'identifiers' | 'type'>[];
+  auth0Handlers: (Pick<APIHandler, 'id' | 'type'> & {
+    identifiers: (string | string[])[];
+  })[];
```

Comment:
There is a morass of logic elsewhere in the codebase that made it untenable to modify the type directly on the `APIHandler`'s `identifiers` field.